### PR TITLE
remove python3.4 support

### PR DESCRIPTION
**Pull request by NotAFile**
also bump twisted version

### CODE REVIEW

**Comment by feikname** [comment]
Why was py3.7 removed from travis for now?

**Comment by NotAFile** [comment]
travis, surprisingly, doesn't support it properly yet

File: scripts/_docker_build.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ set -e -x
 # code from https://github.com/pypa/python-manylinux-demo/blob/master/travis/build-wheels.sh
 # Compile wheels
 cd /io/
-for PYBIN in /opt/python/{cp27-cp27m,cp27-cp27mu,cp34-cp34m,cp35-cp35m,cp36-cp36m}/bin; do
+for PYBIN in /opt/python/{cp35-cp35m,cp36-cp36m}/bin; do
```
Comment (swalladge) [inline scripts/_docker_build.sh]: do we need to add an entry similar to `cp36-cp36m` for python37?

Comment (swalladge) [reply scripts/_docker_build.sh]: done in #392

**Pull request merged**